share/misc/bsd-family-tree

```diff
@@ -447,39 +447,45 @@ FreeBSD 5.2           |      |                 |                       |
  |  FreeBSD           |      |                    |                    |
  |   13.2             |      |                    |                    |
  |                    |      |                    |                    |
-FreeBSD 14 -current   |  NetBSD -current   OpenBSD -current    DragonFly -current
+ *--FreeBSD           |      |                    |                    |
+ |   14.0 prerelease  |      |                    |                    |
+ |                    |      |                    |                    |
+FreeBSD 15.0-CURRENT  |  NetBSD-current  OpenBSD-current  DragonFly-DEVELOPMENT
  |                    |      |                    |                    |
  v                    v      v                    v                    v
 
 Time
 ----------------
 
-Time tolerance +/- 6 months, depending on which book/article you read; if it
-was the announcement in Usenet or if it was available as tape.
+Time tolerance on 4.3BSD and earlier releases can be as large as six months due
+to the uncertainty of historical records; the early release to special groups;
+and the then slowness of tape duplication. Articles and books from the period
+have different dates, since they relied on different sources: newest file in
+the distribution vs public announcement vs first available tape.
```

Comment:
"In some cases the differences have been noted in the entry."

Reply:
> In some cases the differences have been noted in the entry.

Add it to what's drafted (a two-sentence paragraph), to create a third sentence? 

Or use it to replace the second sentence?

Currently in `main`:

```text
Time tolerance +/- 6 months, depending on which book/article you read; if it
was the announcement in Usenet or if it was available as tape.
```